src/components/regions/ProbeDetails.svelte

```diff
@@ -279,15 +282,16 @@ h2 {
             Documentation
           </a>
         </div>
-    </div>
-    <!-- /probe-details-download -->
+      </div>
+      <!-- /probe-details-download -->
 
-</div>
-{:else}
+    </div>
+  {:else}
     <div class="drawer-section">
-        <div class="empty-details">
-            search for a telemetry probe above
-        </div>
+      <div class="empty-details">
+        search for a telemetry probe above
+      </div>
     </div>
-{/if}
+  {/if}
 </div>
+<!-- /details-drawer -->
```

Comment:
I'm not sure if I understand what this comment means. Does it mean that the `div` on the line above closes `.details-drawer`? Maybe we could add it after the `</div>` on the same line.

Reply:
There's a decent amount of markup in this component so just some markers to delineate tag blocks once you scroll too far down. The compiler will remove these in prod.

Reply:
No problem. If anything, maybe it would make sense to move it to the line above? I've seen and used these comments, but I think I was confused because I've never seen it on the *next* line.